src/views/TechnicalAssistance/RequestForm/SubmittedRequest.tsx

```diff
@@ -0,0 +1,295 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Grid } from '@trussworks/react-uswds';
+import { camelCase, capitalize } from 'lodash';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import Divider from 'components/shared/Divider';
+import useTRBAttendees from 'hooks/useTRBAttendees';
+import {
+  GetTrbRequest_trbRequest as TrbRequest,
+  GetTrbRequest_trbRequest_form as TrbRequestForm
+} from 'queries/types/GetTrbRequest';
+import { TRBWhereInProcessOption } from 'types/graphql-global-types';
+import { formatDateLocal, formatDateUtc } from 'utils/date';
+
+import { AttendeesTable } from './AttendeesForm/components';
+import DocumentsTable from './DocumentsTable';
+
+function SubjectDefinition(
+  form: any,
+  field: keyof TrbRequestForm
+): React.ReactNode {
+  const { t } = useTranslation('technicalAssistance');
+  return Array.isArray(form[field]) && form[field].length ? (
+    form[field]
+      .map((v: string) =>
+        v === 'OTHER'
+          ? `${t('basic.options.other')}: ${form[`${field}Other`]}`
+          : t(`subject.options.${field}.${v}`)
+      )
+      .join(', ')
+  ) : (
+    <em>{t('check.noTopicsSelected')}</em>
+  );
+}
+
+type SubmittedRequestProps = {
+  request: TrbRequest;
+  showEditSectionLinks?: boolean;
+};
+
+function SubmittedRequest({
+  request,
+  showEditSectionLinks
+}: SubmittedRequestProps) {
+  const { t } = useTranslation('technicalAssistance');
+
+  const {
```

Comment:
In the Figma, the requester is shown first and then the rest of the attendees. You can get the requester like this:

```
const {
  data: { requester, attendees }
} = useTRBAttendees(request.id, request.createdBy);
```

Passing the requester EUA as the second parameter in useTRBAttendees makes sure that it uses that EUA to get the requester attendee object vs the current user.

The Figma also displays the requester's role as "Requester, Role" instead of just their role like the rest of the attendees. That and the table sort order might take a bit of refactoring on the `<AttendeesTable />` component, so it might be worth just adding the requester to the table for now and refactoring the table later?

Sorry for not catching this sooner!!

Reply:
No problem. Change is in